Dockerfile.misc

```diff
@@ -44,7 +44,8 @@ RUN apt update -y -q && apt upgrade -y -q && apt update -y -q && \
     re2c \
     perl \
     cpanminus \
-    openssh-client 
+    openssh-client \
```

Comment:
Just for context: I've been trying to avoid adding more and more stuff to this Dockerfile; though this OK for this size of project. The newer "misc" stuff all has its own Docker file and build scripts etc separately so we don't just keep making this one bigger and bigger and we never quite know why we have all these dependencies.